pkg/kubectl/cmd/describe.go

```diff
@@ -59,7 +59,7 @@ $ kubectl describe po -l name=myLabel`,
 
 func RunDescribe(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []string) error {
 	selector := cmdutil.GetFlagString(cmd, "selector")
-	cmdNamespace, err := f.DefaultNamespace()
+	cmdNamespace, _, err := f.DefaultNamespace()
```

Comment:
I don't think they can, at least not any way that I saw.

```
Usage: 
  kubectl describe (RESOURCE NAME | RESOURCE/NAME) [flags]
```

So I ignored the overridden bool when the builder didn't have a `FilenameParam` call on it to specify files.

Am I missing something here?